### ISSUE TITLE
0.13.x kalite commands do not resolve paths with specific user names

### ISSUE THREAD

**Issue by EdDixon** [learningequality/ka-lite#3530]
Many of the related kalite command scripts such as kalite start rely on env paths and system paths, then building complete paths out from those. The current scripts are not allowing the use of certain systems usernames on Windows which are required by the scripts to create the internal paths needed. Currently the restriction is that "User Name must contain only letters, digits, and underscores, and start with a letter." So a user name with first and last will not resolve etc. @MCGallaspy and @benjaoming had previously resolved this issue by using proper escape characters in usability tests months ago but those may not have been merged into 0.13.x This can be replicated by running with a non compliant user name and then running the bat files directly as shown in the screen capture.
![screenshot from 2015-04-16 16 34 21](https://cloud.githubusercontent.com/assets/4349806/7194018/8469385e-e465-11e4-84f0-0b16a947f0dd.png)

**Comment by MCGallaspy** [comment]
This got lost in the shuffle. @ed are you still having trouble with the `kalite` command with usernames that have a space? Can you list specific commands which fail, and the error you received? From your screenshot, I'm guessing this specific error is from the `setup` management command, specifically line 243 (in `develop`):
```python
username = options["username"] or getattr(settings, "INSTALL_ADMIN_USERNAME", None)
```

**Comment by aronasorman** [comment]
Fixed in #3922, it seems.
